data/test/lua/wml_tags.lua

```diff
@@ -0,0 +1,33 @@
+--! #textdoman wesnoth-test
+
+local T = wml.tag
+
+function wesnoth.wml_actions.test_do_attack_by_id(cfg)
+	if not cfg.source then
```

Comment:
This attribute has to be renamed, because wmllint expects `source` to have a translation mark.